Description: Fix race between sddm and logind
 Make sure sddm is run after logind. An explicit dependency will make sddm run
 after logind every time.
Author: Pier Luigi Fiorini <pierluigi.fiorini@liri.io>
Origin: upstream
Bug: https://github.com/sddm/sddm/issues/978
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1763977
Applied-Upstream: commit:fca8c80
Last-Update: 2018-04-15
--- a/services/sddm.service.in
+++ b/services/sddm.service.in
@@ -2,7 +2,7 @@
 Description=Simple Desktop Display Manager
 Documentation=man:sddm(1) man:sddm.conf(5)
 Conflicts=getty@tty1.service
-After=systemd-user-sessions.service getty@tty1.service plymouth-quit.service
+After=systemd-user-sessions.service getty@tty1.service plymouth-quit.service systemd-logind.service
 
 [Service]
 ExecStart=@CMAKE_INSTALL_FULL_BINDIR@/sddm
